FrontEnd/src/components/ProfilePage/FormComponents/FormFields/ImageField.jsx

```diff
@@ -2,62 +2,83 @@ import classNames from 'classnames';
 import preventEnterSubmit from '../../../../utils/preventEnterSubmit';
 import css from './ImageField.module.css';
 
-const ImageField = (props) => {
+const ImageField = ({
+  name,
+  label,
+  accept,
+  inputType = 'text',
+  value,
+  updateHandler,
+  onDeleteImage,
+  error,
+}) => {
   const backgroundImage = {
-    background: `url(${props.value}) lightgray 50% / cover no-repeat`,
+    background: `url(${value}) lightgray 50% / cover no-repeat`,
   };
 
-  return (
-    <div
-      className={classNames(css['fields__column'], {
-        [css['fields__column--logo']]: props.name === 'logo_image',
-      })}
+  const renderInput = () => (
+    <input
+      accept={accept}
+      id={name}
+      type={inputType}
+      className={css['upload-file__input']}
+      name={name}
+      onChange={updateHandler}
+      onKeyDown={preventEnterSubmit}
+    />
+  );
+
+  const renderUpdateImageLabel = (text) => (
+    <label className={css['update-file__label']} htmlFor={name}>
+      <img
+        className={css['upload-file__icon']}
+        src={`${process.env.REACT_APP_PUBLIC_URL}/profilepage/camera_icon.png`}
+        alt="Change icon"
+      />
+      <span className={css['update-file__text']}>{text}</span>
+    </label>
+  );
+
+  const renderDeleteButton = (text) => (
```

Comment:
nit: Not sure whether these styles are used anywhere, but it still seems that it's better to use component instead if function.

Reply:
I suppose such approach is only used in this ImageField component. I'll consider about moving such functions into a separate component.